source/_integrations/tplink.markdown

```diff
@@ -35,50 +38,87 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
-## Supported Devices
-### Plugs
-
-- HS100
-- HS103
-- HS105
-- HS110 (supports consumption sensors)
-- KP105
-- KP115 (supports consumption sensors)
+## TP-Link Kasa Device Overview
 
-### Strip (Multi-Plug)
+### Plugs
 
-- HS107 (indoor 2-outlet)
-- HS300 (powerstrip 6-outlet) (supports consumption sensors)
-- KP303 (powerstrip 3-outlet)
-- KP400 (outdoor 2-outlet)
-- KP200 (indoor 2-outlet)
-- KP40 (outdoor 2-outlet)
-- EP40 (outdoor 2-outlet)
+| Model Number     | Support by Home Assistant | Support by TP-Link's 'local only mode'     | Note                                     |
```

Comment:
Please avoid tables, these are nearly impossible to view nicely on a mobile device.

Reply:
What would you recommend as an alternative formatting?

Reply:
Anything but tables :)

Reply:
I mean, the file already contains tables plus they really don't look *that* bad. I'm not sure there's a cleaner way to display this amount of information.

You're sure that's not just a personal preference? 🤔 

![IMG_20220502_133803](https://user-images.githubusercontent.com/614929/166227901-f9d8d8c0-c3a0-49ba-b74d-25e072bc4192.jpg)

Reply:
We generally ask to avoid those, I think it is weird to make an exception in this case.

Thanks 👍

Reply:
Ah okay - didn't know this. Is there a style guideline or something like that for the docs? :)

Reply:
I was looking through the other integrations to see how this is done for other pages, and there are quite a few using tables as well:

- yeelight
- xiaomi_miio
- philips_js
- zha

Feels more like the norm than an exception :)

Reply:
Please remove the use of tables as much as possible.

Reply:
I think the simplest way to remove the tables is to not enumerate which devices support local only mode and instead link to Tplink's article.  We really don't want to get issue reports for incorrect documentation because we have copied tplink's incorrect documentation (I know at least the KL420 requires a kasa account even though their docs don't mention that).  In short we only need to know what Home Assistant supports for Home Assistant's documentation.

AFAICT all the devices that removed the local api when linking to cloud have the option to restore it by following the procedure